src/functions/negate.js

```diff
@@ -3,13 +3,39 @@ import {wrap} from "../core/wrap";
 // Get the complement of a function.
 export const negate = wrap({
     name: "negate",
+    summary: "Get the logical negation of a predicate function.",
+    docs: process.env.NODE_ENV !== "development" ? undefined : {
+        expects: (`
+            The function expects a single predicate function as its input.
+        `),
+        returns: (`
+            The function builds and returns a function returning true for
+            which inputs producing a falsey value when passed to the
+            inputted predicate and returning false for all inputs producing
+            a truthy value when passed to that predicate.
+        `),
+        examples: [
+            "basicUsage"
+        ],
+        related: [
+            "allPass", "anyPass", "nonePass"
+        ],
+    },
     attachSequence: false,
     async: false,
     arguments: {
         one: wrap.expecting.function
     },
     implementation: (predicate) => {
-        return (...args) => (!prediate(...args));
+        return (...args) => (!predicate(...args));
+    },
+    tests: process.env.NODE_ENV !== "development" ? undefined : {
+        "basicUsage": hi => {
+            const even = (a) => (a % 2 === 0);
+            const odd = hi.negate(even);
+            hi.assert(even(2));
+            hi.assert(odd(3));
```

Comment:
One next step might be to grab these salient lines using `tests.x.toString()` and shove them into the `docs` object. It would be good to get the source as exactly represented here though, e.g. currently if I do that I then get this back

``` js
function basicUsage(hi) {
            var even = function even(a) {
                return a % 2 === 0;
            };
            var odd = hi.negate(even);
            hi.assert(even(2));
            hi.assert(odd(3));
        }
```

Comments do seem to be preserved though so if this is the worst rewriting that happens I don't think this is unacceptable, just slightly annoying.